src/client/createDomElem.js

```diff
@@ -0,0 +1,22 @@
+const NS = [
+  'http://www.w3.org/1999/xhtml',
+  'http://www.w3.org/2000/svg',
+]
+
+const elements = {}
+
+export default (name: string): HTMLElement | Element => {
+  if (name in elements) return elements[name]
+
+  let domElem = window.HTMLUnknownElement
+  for (let i = 0; i < NS.length; i += 1) {
+    domElem = document.createElementNS(NS[i], name)
```

Comment:
I couldn't find any support information about this method, if its not widely supported we can't use it and the whole detection is useless.